pdns/pkcs11signers.cc

```diff
@@ -211,11 +213,11 @@ class P11KitAttribute {
 
 class Pkcs11Slot {
   private:
-    bool d_logged_in;
+    bool d_logged_in{};
```

Comment:
I wouldn't mind setting `d_logged_in` and `d_err` to `false` explicitly. The default-initialization of integral types should be zero-initialization, but I always have to think twice (at least!) to be sure.